### PR TITLE
dev-libs/libuv: Bump to 1.41.0

### PR DESCRIPTION
**Pull request by jsmolic**
* Re-enable test suite
* Drop static-libs support
* Add live ebuild template

### CODE REVIEW

File: dev-libs/libuv/libuv-9999.ebuild [hunk 1]
```diff
@@ -1,49 +1,67 @@
...
 
 multilib_src_configure() {
-	local myeconfargs=(
+	ECONF_SOURCE="${S}" \
+	econf \
+		--disable-static \
 		cc_cv_cflags__g=no
-		$(use_enable static-libs static)
-	)
-	ECONF_SOURCE="${S}" econf "${myeconfargs[@]}"
```
Comment (a17r) [inline dev-libs/libuv/libuv-9999.ebuild:37]: I much prefer `myeconfargs` array style, please keep existing style.

File: dev-libs/libuv/libuv-9999.ebuild [hunk 2]
```diff
@@ -1,49 +1,67 @@
...
 SLOT="0/1"
-KEYWORDS=""
-IUSE="static-libs"
-RESTRICT="test"
 
 BDEPEND="
 	sys-devel/libtool
-	virtual/pkgconfig
-"
+	virtual/pkgconfig"
```
Comment (a17r) [inline dev-libs/libuv/libuv-9999.ebuild:24]: Unnecessary change, please keep existing style.

File: dev-libs/libuv/libuv-9999.ebuild [hunk 3]
```diff
@@ -1,49 +1,67 @@
...
+			src/unix/darwin-proctitle.c || die
+		sed -i -e '/int uv_if_indextoname/i\
+		static size_t strnlen(char *x, size_t len) {\
+			char *r = memchr(x, '"'"'\\n'"'"', len);\
+			if (r == NULL)\
+				return len;\
+			else\
+				return r - x;\
+		}' \
+			src/unix/getaddrinfo.c || die
```
Comment (a17r) [inline dev-libs/libuv/libuv-9999.ebuild:45]: This is ugly to say the least. Please turn it into a patch.

**Review by a17r — CHANGES_REQUESTED** [review]
lgtm except for the nitpicks

File: dev-libs/libuv/libuv-9999.ebuild (hunk 1 above)
Comment (jsmolic) [reply dev-libs/libuv/libuv-9999.ebuild]: Sure, no problem

File: dev-libs/libuv/libuv-9999.ebuild (hunk 2 above)
Comment (jsmolic) [reply dev-libs/libuv/libuv-9999.ebuild]: Done

File: dev-libs/libuv/libuv-9999.ebuild (hunk 3 above)
Comment (jsmolic) [reply dev-libs/libuv/libuv-9999.ebuild]: Agreed, done

**Review by a17r — APPROVED** [review]
Thanks, I can merge this. For the future, I would recommend working on the live ebuild, then do a clean version bump; so the commit log would be something like:
- add release/live switch
- re-enable test suite
- drop static-libs
- 1.41.0 version bump

This makes an isolated look at the live ebuild git log a lot more productive than a bunch of 'sync live ebuild' summaries. If a 9999 version is present, we will always use it to create the next release version, not the other way round.

Patches, unless really trivial, should always contain git format-patch style header with a short description what they do.

Package builds, tests pass.
